src/main/java/com/datastax/oss/cass_stac/entity/Item.java

```diff
@@ -36,14 +36,14 @@ public Double getCloudCover() throws JsonProcessingException {
 
         Map<String, Object> map = objectMapper.readValue(properties, new TypeReference<>() {
         });
-        Object value = map.get("eo:cloud_cover");
-        if (value != null) {
-            try {
-                return Double.parseDouble(value.toString());
-            } catch (NumberFormatException e) {
-                return Double.longBitsToDouble(Long.getLong(value.toString()));
-            }
+        if (properties == null) {
```

Comment:
surpassing empty properties, defaulting value to -1 as it seems like cloud cover is a percentage so -1 will not be included in the range